main.cpp

```diff
@@ -1034,45 +1057,49 @@ void parseCommand()
 
 void sendFeedback()
 {
-    if (main_loop_counter % 50 == 0) {    // Send data periodically
-        if(UART_DMA_CHANNEL->CNDTR == 0) {
-            feedback.start = Feedback::VALID_HEADER;
+    if(UART_DMA_CHANNEL->CNDTR == 0)
```

Comment:
Consider converting this to a guard style if:
```suggestion
    if (UART_DMA_CHANNEL->CNDTR == 0)
        return;
```